### ISSUE TITLE
[BUG] Impossible to edit a document after a migration from paperless-ng, `remove_inbox_tags` is null

### ISSUE THREAD

**Issue by AlexandreBonneau** [paperless-ngx/paperless-ngx#5961]
### Description

I just upgraded from paperless-ng to paperless-ngx, following the guide and changing the image in docker-compose.yml.

Everything works fine, my users, documents and tags are all here.
I can edit/add tags from the document page, however, when I try to edit/add tags, or just "save" any change from that page, I get the following error:
```
{"headers":{"normalizedNames":{},"lazyUpdate":null},"status":400,"statusText":"Bad Request","url":"http://localhost:37007/api/documents/91/","ok":false,"name":"HttpErrorResponse","message":"Http failure response for http://localhost:37007/api/documents/91/: 400 Bad Request","error":{"remove_inbox_tags":["Ce champ ne peut être nul."]}}
```
(where `"Ce champ ne peut être nul."` can be translated to `This field cannot be null`)

What is this `remove_inbox_tags` field?### 

### Steps to reproduce

1. Click "Documents"
2. Edit any field
3. Click "Save"
-> The error is shown, and the document changes are not saved

### Webserver logs

```bash
I'm not sure where I can access those log from the docker `paperless_webserver_1` webserver
```

### Browser logs

```bash
The `paperless.log` file does not show anything related to the error I get, the last line it shows at the time I generated the error was:

[2024-03-01 08:29:54,003] [DEBUG] [paperless.matching] DocumentType Facture matched on document 2023-11-14 Relance sur échéancier because it contains this word: Facture
```
```


### Paperless-ngx version

2.5.4

### Host OS

Debian trixie

### Installation method

Docker - official image

### Browser

Firefox 124.0b5 (64 bits)

### Configuration changes

_No response_

### Other

_No response_

### Please confirm the following

- [X] I believe this issue is a bug that affects all users of Paperless-ngx, not something specific to my installation.
- [X] I have already searched for relevant existing issues and discussions before opening this report.
- [X] I have updated the title field above with a concise description.

**Comment by shamoon** [comment]
Can you try emptying your browser cache and closing any open documents and try again?

**Comment by AlexandreBonneau** [comment]
I opened an private tab, logged in and tried.
The same error is shown.

**Comment by shamoon** [comment]
Ah. Try saving the settings and then try again please.

**Comment by AlexandreBonneau** [comment]
What do you mean by 'saving the settings'? Is that done from within the webapp?

**Comment by shamoon** [comment]
Go to settings and save (you would have to change something and then can change it back of course)

**Comment by AlexandreBonneau** [comment]
I just changed "Add notes", saved, then opened a private tab and logged in again.
Tried to add a tag to a document, and got the error

**Comment by shamoon** [comment]
Odd. How about if you toggle that setting (its under settings) on / off?

(There is a bug here that I have a fix for already, just trying to get your install working again)

**Comment by AlexandreBonneau** [comment]
For info, I was thinking the bug could be related to the document having been created with paperless-ng, not ngx.
So I just scanned a new document, and when trying to edit it, I still get the error.

Do note that add/removing tags from the "Documents" page works as expected.

It gets weirder:
I just tried to toggling the "Activate notes" in the settings and tried to edit the document in between ; still the same error.
However, when I edit this document, I can see the tags that were supposed to not be saved (due to the error).
But the while the tags are shown, the document is not shown in the dashboard view (where it should be shown because I filtered on that tag).
